Features/MultiSelect/test_cases/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from configparser import ConfigParser
+from pathlib import Path
+from common_utilities.fixtures import *
+
+""""This file provides fixture functions for driver initialization"""
+
+global driver
+
+
+@pytest.fixture(scope="session")
+def environment_settings_casesearch():
+    """Load settings from os.environ
```

Comment:
Here it says casesearch but its under multiselect, a bit confusing

Reply:
My intentions just hit the wrong file. 🗡️  Updated in https://github.com/dimagi/dimagi-qa/pull/209/commits/0634ed543e3e5846a8be5e8b7292a2e6a8f513b2